### ISSUE TITLE
missing pokemon to advance but all caught[BUG] 

### ISSUE THREAD

**Issue by raph397** [pokeclicker/pokeclicker#2410]
**Info**
 windows 11
 - Browser Chrome
 - Version 0.9.5

**Description:**
The professor is saying that i'm missing 2 pokemons in Hoenn, when i'm only missing Latias. I looked on the wiki pag and the one that says I am missing is kecleon, but when I encounter one it appears as already caught.

![image](https://user-images.githubusercontent.com/107160579/172746276-f7eec056-88a6-4150-91db-641e9ee41c67.png)

**Comment by RedSparr0w** [comment]
Please fill in all the required information, then feel free to re-open.
Also if you're in Johto, chances are you're missing Celebi.

**Comment by RedSparr0w** [comment]
Have you also caught Deoxys?
The Pokédex only shows up to the highest ID you have encountered.

**Comment by raph397** [comment]
Yes, i have caught deoxys, if u look at the bottom right of the screenshot, pokemon are listed  by name so it should be there

**Comment by xslk** [comment]
If you open up your pokedex and filter by uncaught, what numbers are missing?

**Comment by raph397** [comment]
![image](https://user-images.githubusercontent.com/107160579/172746879-137732ef-2b63-4be9-8cbe-00537e08851b.png)
![image](https://user-images.githubusercontent.com/107160579/172746918-e2c5662e-bef7-43fb-9abd-470eed416a64.png)

**Comment by raph397** [comment]
![image](https://user-images.githubusercontent.com/107160579/172747035-ae2a10eb-11f6-4ab4-834d-0eafbb714d83.png)
oh wow! i didn't know i could see it this way! Thanks!
